site/src/components/SignInForm/SignInForm.tsx

```diff
@@ -119,13 +143,30 @@ export const SignInForm: FC<SignInFormProps> = ({
         </div>
       </form>
       {authMethods?.github && (
-        <div className={styles.submitBtn}>
-          <Link href={`/api/v2/users/oauth2/github/callback?redirect=${encodeURIComponent(redirectTo)}`}>
-            <Button disabled={isLoading} fullWidth type="submit" variant="contained">
-              {Language.githubSignIn}
-            </Button>
-          </Link>
-        </div>
+        <>
+          <div className={styles.divider}>
+            <div className={styles.dividerLine} />
+            <div className={styles.dividerLabel}>Or sign in with</div>
```

Comment:
What does this look like with `or`? `sign in` is a bit duplicative, but it's not bad.

Reply:
<img width="451" alt="Screen Shot 2022-05-31 at 13 04 05" src="https://user-images.githubusercontent.com/3165839/171218907-6bd2ee39-67c7-4791-baaa-77eb4580456d.png">
<img width="458" alt="Screen Shot 2022-05-31 at 13 04 23" src="https://user-images.githubusercontent.com/3165839/171218912-fa261922-b34f-43b0-9589-be69ab4b94b6.png">

Reply:
Feel free to suggest a diff copy for this directly on GitHub so I can just merge it in one click.

Reply:
Good idea!